frontend/src/pages/RunDetails.tsx

```diff
@@ -200,16 +200,21 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
     const runIdFromParams = this.props.match.params[RouteParams.runId];
     return {
       actions: buttons
-        .retryRun(
-          () =>
-            this.state.runMetadata
-              ? [this.state.runMetadata!.id!]
-              : runIdFromParams
-              ? [runIdFromParams]
-              : [],
-          true,
-          () => this.retry(),
-        )
+        // BUG(talebz): AIP-6692 WFSDK: Disable Retry button
+        // .retryRun(
+        //   () =>
+        //     this.state.runMetadata
+        //       ? [this.state.runMetadata!.id!]
+        //       : runIdFromParams
+        //       ? [runIdFromParams]
+        //       : [],
+        //   true,
+        //   () => this.retry(),
+        // )
+        // .newRunFromPipelineVersion(
+        //   () => this.state.runMetadata?.id!,
+        //   () => this.state.runMetadata?.ver!,
+        // )
```

Comment:
Is `.newRunFromPipelineVersion` in comment still needed?

Reply:
done